pkg/kubectl/cmd/create_pdb.go

```diff
@@ -60,7 +60,8 @@ func NewCmdCreatePodDisruptionBudget(f cmdutil.Factory, cmdOut io.Writer) *cobra
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.PodDisruptionBudgetV1GeneratorName)
-	cmd.Flags().String("min-available", "1", i18n.T("The minimum number or percentage of available pods this budget requires."))
+	cmd.Flags().String("min-available", "", i18n.T("The minimum number or percentage of available pods this budget requires."))
```

Comment:
We have done this once before in the `kubectl expose` command (https://github.com/kubernetes/kubernetes/pull/11397). That example may be too old because today we are using a slightly different type of generator (StructuredGenerator) but the concept should be the same: introduce a second generator, continue defaulting in the first generator that sets minAvailable=1. We can also add a notice of deprecation for the old generator and switch to the new generator in the next release I think.